### PR TITLE
add twitter pixel

### PR DESCRIPTION
**Pull request by jasonbriggs**

### CODE REVIEW

File: layouts/partials/twitterPixel.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!-- Twitter universal website tag code -->
+<script src="//platform.twitter.com/oct.js" type="text/javascript"></script>
```
Comment (kujenga) [inline layouts/partials/twitterPixel.html]: Can we make this always https?

Comment (jasonbriggs) [inline layouts/partials/twitterPixel.html]: yeah good call!

**Comment by jasonbriggs** [comment]
I'll wait on this until Craig figures out what is required for AdWords

**Comment by jasonbriggs** [comment]
This should be good to go now

File: layouts/partials/adwordsPixel.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+var google_conversion_id = 879461873;
+var google_custom_params = window.google_tag_params;
+var google_remarketing_only = true;
+/* ]]> */
+</script>
+<script type="text/javascript" src="//www.googleadservices.com/pagead/conversion.js">
+</script>
+<noscript>
+<div style="display:inline;">
+<img height="1" width="1" style="border-style:none;" alt="" src="//googleads.g.doubleclick.net/pagead/viewthroughconversion/879461873/?value=0&amp;guid=ON&amp;script=0"/>
```
Comment (kujenga) [inline layouts/partials/adwordsPixel.html]: Can we set these to https as well?

File: layouts/partials/twitterPixel.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<!-- Twitter universal website tag code -->
+<script src="https://platform.twitter.com/oct.js" type="text/javascript"></script>
+<script type="text/javascript">twttr.conversion.trackPid('nv2mm', { tw_sale_amount: 0, tw_order_quantity: 0 });</script>
+<noscript>
+<img height="1" width="1" style="display:none;" alt="" src="https://analytics.twitter.com/i/adsct?txn_id=nv2mm&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0" />
+<img height="1" width="1" style="display:none;" alt="" src="//t.co/i/adsct?txn_id=nv2mm&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0" />
```
Comment (kujenga) [inline layouts/partials/twitterPixel.html]: https here

File: layouts/partials/adwordsPixel.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!-- Google Code for Remarketing Tag -->
+<script type="text/javascript">
+/* <![CDATA[ */
+var google_conversion_id = 879461873;
+var google_custom_params = window.google_tag_params;
+var google_remarketing_only = true;
+/* ]]> */
+</script>
+<script type="text/javascript" src="//www.googleadservices.com/pagead/conversion.js">
```
Comment (kujenga) [inline layouts/partials/adwordsPixel.html]: https here as well

**Comment by kujenga** [comment]
Setting everything to HTTPS makes configuring the Content-Security-Policy header more straightforward

**Comment by kujenga** [comment]
Great! 👍

**Pull request merged**